packages/x/templates/blueprint/partials/formXElement.tpl

```diff
@@ -108,12 +110,18 @@
             {{/ if }}
             placeholder={t('{{ title }}')}
           >
-            {{# each enumValues }}
-              <Ant.Select.Option value="{{ value }}" key="{{ value }}">{{ label }}</Ant.Select.Option>
-            {{/ each }}
+          {
+            options.map(
+              (option)=>(
+              <Ant.Select.Option value={option.value} key={option.value} >
```

Comment:
@y-moufakkr indent this properly I know it's a tpl but some tabs will still be useful and spacing